test/features/service_discovery.feature

```diff
@@ -54,4 +54,51 @@ Feature: Discovery with onboarding
     @native
     Examples: Native
       | native  | minutes |
-      | enabled | 20      |
\ No newline at end of file
+      | enabled | 20      |
+
+#####
+
+  Scenario Outline: Deploy Spring Boot onboarding example
```

Comment:
This can be transformed to scenario as there are no parameter combinations to be executed.

Reply:
Changed and tested again in Openshift.